docs/source/contributing/developing.adoc

```diff
@@ -280,6 +280,17 @@ For example, tests can be run against a stopped {project} instance with the imag
 $ make integration_all RUN_BEFORE_FEATURE="start; stop; config set image-caching true"
 ----
 
+COPY_OC_FROM::
+
+To test Minishift with non-default oc binary you can use the `COPY_OC_FROM` parameter.
+It specifies the path to the oc binary which will be copied to the Minishift home folder before every feature run.
+Please note that the `oc` binary must be contained in the same folder structure which Minishift uses for storage of oc binaries, eg.: `<version>/<platform>/oc(.exe)`.
+For example to test Minishift with custom build of `oc` in version `v3.7.23` oc run:
```

Comment:
I'm not sure what this sentence actually means or is trying to say. The wording of "custom build of `oc` in version `v3.7.23` oc run" is unclear.

Reply:
@robin-owen I agree. Made it more simple, hopefully it will be understandable now.